src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions.mdx

```diff
@@ -177,7 +177,9 @@ Creating a buffer:
 
   You're creating an alert condition to notify your team of any latency issues with the `WebPortal` application. In this example, your application consistently sends New Relic data. There is a constant stream of signals being sent from your application to New Relic, and there is no expected gap in signal, so you won't need to select a gap-filling strategy.
 
-  Gap-filling strategies address scenarios where data collection might be intermittent or incomplete. They provide a method for substituting missing data points with estimated values, ensuring that alert conditions can still function effectively even with gaps in the data stream.
+  Once you set up an alert condition, the entity status color will change to green to let you know that it's being monitored. If the entity status color is gray that can mean a few things: your entity is not being monitored, there was a loss of signal, or the data collection might be intermittent or incomplete. Some customers like to use gap-filling strategies to make sure that, in the case of incomplete or intermittent data, the entity status remains green. 
```

Comment:
"will change" --> "changes"

Reply:
Although I like the impulse behind the "Some customers" sentence. I think in this case, I'd prefer something a little more direct:

"Use gap-filling strategies to make sure that, even in the case of incomplete or intermittent data, your entity status stays green."

Reply:
Fixed!

Reply:
I wanted to say some customers because it's expensive to do this so I didn't want to recommend it full stop. What do you think of my change?